flow/project.py

```diff
@@ -1117,68 +1093,32 @@ def _incomplete(s):
             print(json.dumps(statuses, indent=4), file=file)
             return
 
-        # Generate status overview:
         if overview:
-            print("# Overview:", file=file)
-            print("{} {}\n".format(self._tr("Total # of jobs:"), len(statuses), file=file))
-
-            # Draw progress bars
+            # get overview info:
+            column_width_bar = 50
+            column_width_label = 5
+            for key, value in self._label_functions.items():
+                column_width_label = max(column_width_label, len(key.__name__))
             progress = defaultdict(int)
             for status in statuses.values():
                 for label in status['labels']:
                     progress[label] += 1
             progress_sorted = list(islice(
                 sorted(progress.items(), key=lambda x: (x[1], x[0]), reverse=True),
                 overview_max_lines))
-            rows = [[
-                label,
-                '{} {:0.2f}%'.format(draw_progressbar(num, len(statuses)),
-                                     100 * num / len(statuses))
-            ]
-                for label, num in progress_sorted]
-
-            print(tabulate.tabulate(rows, headers=['label', 'ratio']), file=file)
-            if not rows:
-                print("[no labels to show]", file=file)
-            if overview_max_lines is not None:
-                lines_skipped = len(progress) - overview_max_lines
-                if lines_skipped > 0:
-                    print(self._tr("Lines omitted:"), lines_skipped, file=file)
-
-        # Generate detailed view:
-        def _select_op(doc):
-            active = JobStatus(doc['scheduler_status']) > JobStatus.unknown
-            if skip_active and active:
-                return False
-            if not all_ops and not (active or doc['eligible']):
-                return False
-            return True
```

Comment:
How is skip_active option passed to the function? the status command don't have this option and I am confused what will trigger skip_active

Reply:
The `--skip-active` option is active, but not shown in the help function, probably because it is supposed to be deprecated, but we forgot to put a deprecation warning into the code.
https://github.com/glotzerlab/signac-flow/blob/f0b696c11a41700ae0bc50f608d5a7a1b10a442e/flow/project.py#L2021-L2024